hybrid/decomposers.py

```diff
@@ -60,24 +60,24 @@ class EnergyImpactDecomposer(Runnable, traits.ProblemDecomposer):
             by at least this amount.
 
         rolling (bool, optional, default=True):
-            Should successive calls for the same problem (but maybe different samples)
-            produce subproblems on different variables rolling down the list of all variables
-            sorted by decreasing impact?
+            If True, successive calls for the same problem (with possibly different samples)
+            produce subproblems on different variables, selected by rolling down the list of all
+            variables sorted by decreasing impact.
 
         rolling_history (float, optional, default=0.1):
-            Size of unrolled variables pool, as a fraction of the problem size (0.0 to 1.0).
-            Determines the reset condition for subproblem unrolling.
+            Ratio, as a float in range 0.0 to 1.0, of variables as yet unselected by the
+            rolling selection to the problem size. Determines the reset condition for subproblem unrolling.
```

Comment:
This is actually wrong. Rolling history size (expressed as a fraction of problem size) determines how many variables should be "unrolled" before resetting. We'll talk offline.